### PR TITLE
Add ES6 Lexical Declarations sample

### PR DESCRIPTION
**Pull request by addyosmani**
reviewer: @jeffposnick

### CODE REVIEW

File: lexical-declarations-es6/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Lexical Declarations (ES6) Sample
+===
+See https://googlechrome.github.io/samples/lexical-declarations-es6/index.html for a live demo.
+
+Learn more at https://www.chromestatus.com/features/4645595339816960
```
Comment (jeffposnick) [inline lexical-declarations-es6/README.md]: Can you link to https://www.chromestatus.com/feature/4645595339816960 (`feature`, not `features`) to go directly to the landing page?

File: lexical-declarations-es6/index.html [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+    <link rel="icon" href="../images/favicon.ico">
+
+    <link rel="stylesheet" href="../styles/main.css">
+</head>
+
+<body>
+<h1>Lexical Declarations (ES6) Sample</h1>
+
+<p>Available in <a href="https://www.chromestatus.com/features/4645595339816960">Chrome 41+</a></p>
```
Comment (jeffposnick) [inline lexical-declarations-es6/index.html]: `features` → `feature`

File: lexical-declarations-es6/index.html [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+<!-- // [START code-block] -->
```
Comment (jeffposnick) [inline lexical-declarations-es6/index.html]: Could you move this down to line 64? If you end up writing an updates.h5r.com article, you probably won't want to pull in the marked-up prose from this page, and will just want to pull in the portion that's "code".

**Comment by jeffposnick** [comment]
Reviewed.

**Comment by addyosmani** [comment]
Comments addressed. Thanks!

**Comment by jeffposnick** [comment]
:+1:

**Pull request merged**